docs/workflow/requirements/windows-requirements.md

```diff
@@ -1,140 +1,109 @@
 
-The _dotnet/runtime_ repository requires using CMake 3.20 or newer.
+- .NET desktop development
+- Desktop development with C++
 
-**NOTE**: If you plan on using the `-msbuild` flag for building the repo, you will need version 3.21 at least. This is because the VS2022 generator doesn't exist in CMake until said version.
+To build the tests and do ARM32/ARM64 development, you'll need some additional components. You can find them by clicking on the *Individual components* tab in the *Visual Studio Installer*:
 
-#### Ninja
+- For ARM stuff: *MSVC v143 - VS 2022 C++ ARM64/ARM64EC build tools (Latest)* for Arm64, and *MSVC v143 - VS 2022 C++ ARM build tools (Latest)* for Arm32.
+- For building tests: *C++/CLI support for v143 build tools (Latest)*
 
-* Install Ninja in one of the three following ways
-  * Ninja is included with Visual Studio. ARM64 Windows should use this method as other options are currently not available for ARM64.
-  * [Download the executable](https://github.com/ninja-build/ninja/releases) and add its location to [the Default PATH variable](#adding-to-the-default-path-variable).
-  * [Install via a package manager](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages), which should automatically add it to the PATH environment variable.
+Alternatively, there is also a `.vsconfig` file included at the root of the runtime repo. It includes all the necessary components required, outlined in a JSON format that Visual Studio can read and parse. You can boot up Visual Studio directly and [import this `.vsconfig` file](https://learn.microsoft.com/visualstudio/install/import-export-installation-configurations?view=vs-2022#import-a-configuration) instead of installing the workloads yourself. It is worth mentioning however, that while we are very careful in maintaining this file up-to-date, sometimes it might get a tad obsolete and miss important components. So, it is always a good idea to double check that the full workloads are installed.
 
-#### Python
+#### Individual Development Tools
 
-* Install [Python](https://www.python.org/downloads/) for Windows.
-* Add its location (e.g. C:\Python*\\) to the PATH environment variable.
-  The installation script has a check box to do this, but you can do it yourself after the fact following the instructions at [Adding to the Default PATH variable](#adding-to-the-default-path-variable).
+All the tools you need should've been installed by Visual Studio at this point. Some of those tools, however, may not have been installed or you might prefer installing them yourself from their own sources. The main process for this is to download their installers and follow their setup. Said installers usually also prompt you to add them automatically to your `PATH` environment variable. If you miss this option, or prefer to set them yourself later on, you can follow the instructions in the [Setting Environment Variables on Windows](#setting-environment-variables-on-windows) section of this doc.
```

Comment:
I am not sure what the "Said" in "Said installers usually also prompt...is refering to...

Reply:
It's referring to the tools' installers that were described in the previous sentence.